schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/errors/InvalidSchemaBranchDeletionException.java

```diff
@@ -25,4 +29,14 @@ public InvalidSchemaBranchDeletionException(String message) {
     public InvalidSchemaBranchDeletionException(String message, Throwable throwable) {
         super(message, throwable);
     }
+
+    public static InvalidSchemaBranchDeletionException getSchemaVersionTiedToOtherBranchException(Map<Integer, List<String>> versionToBranchMap) {
```

Comment:
This does not look to be a common utility used by other parts of the code, better to have this method where this is currently used.